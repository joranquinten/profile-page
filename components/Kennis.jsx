import React from 'react';
import styled from 'styled-components';

import { colors } from '../styled/themes/default';

import Opleiding from './Opleiding';
import Onderwerpen from './Onderwerpen';
import Media from './Media';

const StyledKennis = styled.div`
  .ant-card {
    border-color: ${colors.blackDarkened};
    background: ${colors.blackLightened};
    color: ${colors.white};
    margin: 0 0 2rem;

  .ant-card-head {
    border-color: ${colors.blackDarkened};
  }

  .ant-card-head-title {
    color: ${colors.tertiaryColor};
  }
`;

const Kennis = () => (
  <StyledKennis>
    <Opleiding />
    <Onderwerpen />
    <Media />
  </StyledKennis>
);

export default Kennis;
