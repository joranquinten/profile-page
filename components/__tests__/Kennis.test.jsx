import { shallow } from 'enzyme';

import Kennis from '../Kennis';

jest.mock('../../data/nl.js');

describe('<Kennis />', () => {
  it('should render without crashing', () => {
    expect(shallow(<Kennis />)).toMatchSnapshot();
  });
});
