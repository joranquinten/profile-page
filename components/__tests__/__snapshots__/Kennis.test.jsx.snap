// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Kennis /> should render without crashing 1`] = `
<Kennis__StyledKennis>
  <Opleiding />
  <Onderwerpen />
  <Media />
</Kennis__StyledKennis>
`;
